### PR TITLE
Update debug to maintenance

### PR DESCRIPTION
**Pull request by subhamkrai**
**Description of your changes:**
maintenance: update debug cmd to maintenance cmd

debug command can be confusing for users so after discussion with
team we are changing it to maintenance command.

mons: update restore-quorum logs and docs

since updating debug to maintenance, it requires
changing in the mons restore command as it used debug mode
or maintenance mode for restoring mons.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] CI tests has been updated, if necessary.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.

### CODE REVIEW

File: cmd/commands/maintenance.go [hunk 1]
```diff
@@ -17,44 +17,44 @@ limitations under the License.
...
 )
 
-// OperatorCmd represents the operator commands
-var DebugCmd = &cobra.Command{
-	Use:                "debug",
-	Short:              "Debug a deployment by scaling it down and creating a debug copy. This is supported for mons and OSDs only",
+// MaintenanceCmd represents the operator commands
+var MaintenanceCmd = &cobra.Command{
+	Use:                "maintenance",
+	Short:              "Perform maintenance operation on mons and OSDs deployment by scaling it down and creating a maintenance copy.",
```
Comment (travisn) [inline cmd/commands/maintenance.go:27]:
```suggestion
	Short:              "Perform maintenance operation on mons and OSDs deployment by scaling it down and creating a maintenance deployment.",
```

File: docs/mons.md [hunk 2]
```diff
@@ -42,102 +42,143 @@ kubectl rook-ceph mons restore-quorum c
...
-mon=a, endpoint=192.168.64.169:6789
-Info: Check for the running toolbox
-Info: Waiting for the pod from deployment "rook-ceph-tools" to be running
-deployment.apps/rook-ceph-tools condition met
-
-Warning: Restoring mon quorum to mon c (192.168.64.167)
-Info: The mons to discard are: b a
-Info: Are you sure you want to restore the quorum to mon "c"? If so, enter: yes-really-restore
+# Info: mon c state is leader
+# mon=b, endpoint=192.168.64.168:6789
```
Comment (travisn) [inline docs/mons.md:48]: Do all of these lines have the `#` prefix? I thought some output lines don't have it.

**Review by travisn — CHANGES_REQUESTED** [review]
Looks good, just a couple nits

File: docs/mons.md (hunk 2 above)
Comment (subhamkrai) [reply docs/mons.md:48]: `#` prefix is only to separate output from commands.  We don't add `#`  in front of the command output.

File: cmd/commands/maintenance.go (hunk 1 above)
Comment (subhamkrai) [reply cmd/commands/maintenance.go]: done

File: docs/mons.md (hunk 2 above)
Comment (travisn) [reply docs/mons.md:48]: I'd suggest if there are commands in the same block with the output, the command would have the `$` prefix, while the output is exactly as we see it in the console. I thought this was the pattern for the rook docs, which we could follow here as well.

Comment (subhamkrai) [reply docs/mons.md]: I have removed `#` from the output now and slowly going in the future I feel maintaining the long output will become maintenance burden

File: docs/mons.md [hunk 3]
```diff
@@ -59,85 +60,126 @@ Info: Are you sure you want to restore the quorum to mon "c"? If so, enter: yes-
...
-Error from server (NotFound): persistentvolumeclaims "rook-ceph-mon-b" not found
-Info: purging old mon: a
-deployment.apps "rook-ceph-mon-a" deleted
-Error from server (NotFound): services "rook-ceph-mon-a" not found
-Info: purging mon pvc if exists
-Error from server (NotFound): persistentvolumeclaims "rook-ceph-mon-a" not found
-Info: Mon quorum was successfully restored to mon c
-Info: Only a single mon is currently running
-Info: Press Enter to start the operator and expand to full mon quorum again
+Error: failed to run command. failed to run command. command terminated with exit code 1%!(EXTRA string=failed to get the status of ceph cluster)
```
Comment (travisn) [inline docs/mons.md:116]: Formatting error in the output

File: docs/mons.md [hunk 4]
```diff
@@ -59,85 +60,126 @@ Info: Are you sure you want to restore the quorum to mon "c"? If so, enter: yes-
...
+   data:
+     volumes: 1/1 healthy
+     pools:   4 pools, 113 pgs
+     objects: 9 objects, 580 KiB
+     usage:   32 MiB used, 1.5 TiB / 1.5 TiB avail
+     pgs:     113 active+clean
+
+
+
+ Info: sleeping for 5 seconds
```
Comment (travisn) [inline docs/mons.md:146]:
```suggestion
Info: sleeping for 5 seconds
```

File: docs/mons.md (hunk 2 above)
Comment (travisn) [reply docs/mons.md]: Yeah let's discuss about long term

**Review by travisn — APPROVED** [review]
Just a couple more formatting nits, I'll approve for you to merge after that

File: docs/mons.md (hunk 4 above)
Comment (subhamkrai) [reply docs/mons.md]: removed the extra space

File: docs/mons.md (hunk 3 above)
Comment (subhamkrai) [reply docs/mons.md]: updating [this](https://github.com/rook/kubectl-rook-ceph/pull/252/files#diff-94cb19f5706322197b23e830805b02726b6d7b75ae831c5a7ed19b1a66675a31R275) fixed the issue

**Pull request merged**